Passports please.

Flying-loulou opened the pull request:
:cl: Flying_loulou
rscadd: Adds a selection of passports.
/:cl:

Adds a selection of passports : SCG, SCG member states (Earth, Mars...), ICCG, ICCG Member states (Terra, etc...), and a few independant colonies (Saveel, Mohranda, Magnitka). The have no function other than being here and look fancy (it's an inert RP prop).

See the icon diff for the sprites.
https://www.youtube.com/watch?v=s0vNsH81YeA

PsiOmegaDelta commented on maps/torch/loadout/loadout_accessories.dm:
```diff
@@ -288,3 +288,39 @@
+	passport["Saveel passport"] = /obj/item/weapon/passport/saveel
+	passport["Magnitkan passport"] = /obj/item/weapon/passport/magnitka
+	passport["Mohrandade passport"] = /obj/item/weapon/passport/empiremohranda
+	gear_tweaks += new/datum/gear_tweak/path(passport)
```
Any particular reason that this cannot be `gear_tweaks += new/datum/gear_tweak/path/subtype(/obj/item/weapon/passport)`?

Flying-loulou replied:
none. Should it be changed to what you're proposing ?

PsiOmegaDelta replied:
Even better would be:
```
/datum/gear/passport
	display_name = "passports selection"
	description = "A selection of passports."
	path = /obj/item/weapon/passport
	allowed_branches = CIVILIAN_BRANCHES
	flags = GEAR_HAS_SUBTYPE_SELECTION
```

Flying-loulou replied:
fixed

PsiOmegaDelta commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	desc = "A passport from Luna, within Sol Central Government space."
+
+/obj/item/weapon/passport/scgmember/mars
+	name = "marsian passport"
```
"martian"

Flying-loulou replied:
fixed

PsiOmegaDelta commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	desc = "A passport from Novaya Zemlya, within ICCG space."
+
+/obj/item/weapon/passport/saveel
+	name = "improper Saveel passport"
```
Missing "\\" in "\improper"

Flying-loulou replied:
fixed

Flying-loulou commented:
So I :
- Changed the sprites thanks to bay discord
- Fixed what had to be fixed
- Allowed all branches to use them (no harms in doing so anyway, unless anyone has a _valid_ reason for EC/Fleet to not carry them).

PsiOmegaDelta commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	desc = "A passport from the Earth, within Sol Central Government space."
+
+/obj/item/weapon/passport/scgmember/venus
+	name = "venusian passport"
```
I'm curious, why is this `"venusian passport"` while for example the one above is "\improper Earth passport"?
Seems a bit inconsistent

Flying-loulou replied:
Those with \improper was because I was unsure of the demonym/the demonym felt odd.
earther/earthling ?
Terstener/terstenian ?
Lorrimanian ?

MistakeNot4892 replied:
You don't write 'australian' or 'frenchman' or 'londoner'.

Flying-loulou replied:
> You don't write 'australian' or 'frenchman' or 'londoner'.

I don't get your point.

MistakeNot4892 replied:
They are all proper nouns and should be capitalized.

Flying-loulou replied:
Capitalized it all, poke me if it needs to be changed further. Hopefully it makes sense and is consistent.

Spookerton commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	icon_state = "passport_scg"
+	desc = "A passport from the Sol Central Government."
+
+/obj/item/weapon/passport/scgmember/earth
```
Why `/scgmember/` on all of these?

Flying-loulou replied:
That's a path preference from me, to denote if the faction is independant, member of the SCG or of the ICCG.

MistakeNot4892 replied:
You don't implement any behavior or var changes on `/scgmember`, it doesn't need to exist. Paths aren't tags for coder convenience, they are an inheritance tree.

Flying-loulou replied:
fixed

Spookerton commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	icon_state = "passport_scg2"
+	desc = "A passport from Ceti Epsilon, within Sol Central Government space."
+
+/obj/item/weapon/passport/scgmember/Lordania
```
should lowercase L

Spookerton commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	icon_state = "passport_scg2"
+	desc = "A passport from Lordania, within Sol Central Government space."
+
+/obj/item/weapon/passport/scgmember/Kingston
```
should lowercase K

Spookerton commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	icon_state = "passport_iccg"
+	desc = "A passport from the Independent Colonial Confederation of Gilgamesh."
+
+/obj/item/weapon/passport/iccgmember/gaia
```
Why /iccgmember/ on all of these?

Flying-loulou replied:
fixed

Spookerton commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	desc = "A passport from the southern part of Gaia, under control of the Sol Central Government."
+
+/obj/item/weapon/passport/iccg
+	name = "Independent Colonial Confederation of Gilgamesh passport"
```
`"ICCG passport"` god pls

MistakeNot4892 replied:
Also `\improper` on all of the uppercase-leading names.

MistakeNot4892 replied:
This wasn't resolved.

Spookerton commented on maps/torch/items/items.dm:
```diff
@@ -141,4 +141,143 @@ Weapons
+	hitsound = 'sound/weapons/towelwhip.ogg'
+	desc = "A passport. Its origin seems unkown."
+
+/obj/item/weapon/passport/scg
```
"SCG passport" for conformity with later comment about "ICCG passport"

Flying-loulou replied:
fixed

Spookerton reviewed (changes requested):
You've removed `\improper` from all names - the uppercase ones, so every passport except the base one, should have it so that they are referred to by byond with indefinite articles instead of definite.

Flying-loulou committed:
Adds passports

Flying-loulou commented:
Should be good now.
